# std::errno_of (a polymorphic function)

  the errno associated with an error object

  This field is used in conjunctions with the "IO ERROR" error and represents
  the contents of the *errno* variable when the error object was created.

  Topic: Errors

  See also: std_types::error, std_types::error_number, std::error_category_of,
	    std::error_message_text_of, std::error_details_of,
	    std::failed_attribute_of

  Example:

    load! $buf "some-non-existing-file"
    on buf.is_an_error:
      debug::dump `errno_of(buf)

  Error output:

    errno_of(buf): <error_number 2>

  Example:

    load! $buf "some-non-existing-file"
    on buf.is_an_error:
      println! errno_of(buf).to_integer

  Output:

    2

((defined in runtime_templates/error.template))
((generated by Codestral-22B-v0.1-Q5_K_M.gguf))
((revised))
((2024-07-03 16:52:01))
